cmd/snap/cmd_quota.go

```diff
@@ -165,6 +174,32 @@ func parseCpuQuota(cpuMax string) (count int, percentage int, err error) {
 	return count, percentage, nil
 }
 
+func parseJournalRateQuota(journalRateLimit string) (count int, period time.Duration, err error) {
+	// the rate limit is a string of the form N/P, where N is the number of
+	// messages and P is the period as a time string (e.g 5s)
+	parts := strings.Split(journalRateLimit, "/")
+	if len(parts) != 2 {
+		return 0, 0, fmt.Errorf("missing number of messages and period")
```

Comment:
`"rate limit must be of the form <number of messages>/<period duration>"` ?

Reply:
Yea I agree that is better, thanks!